docs/user/reference/hook.md

```diff
@@ -411,28 +425,37 @@ Workload hooks operate in an environment with additional environment variables a
 
 > [Source](https://github.com/juju/juju/blob/main/internal/charm/hooks/hooks.go)
 
-(hook-action-action)=
-### `<action>-action`
+(hook-action)=
```

Comment:
We should drop the "hook" classification from here too. However, we should find another one. As in, we can't just call these action non-hooks just "actions" because that's the word we use to refer to specific actions from specific charms.